webpack.config.js

```diff
@@ -17,6 +17,11 @@ const webpack = require('webpack');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const WebpackShellPlugin = require('webpack-shell-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const sass = require('sass');
+const crypto = require('crypto');
+
+const crypto_orig_createHash = crypto.createHash;
+crypto.createHash = algorithm => crypto_orig_createHash(algorithm == 'md4' ? 'sha256' : algorithm);
```

Comment:
Linter does not scan this file, but my editor suggests to change `==` to `===` ;)